.github/workflows/deploy-poller.yml

```diff
@@ -23,8 +23,8 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      CHAIN_URL: ${{ vars.CHAIN_URL || github.event.inputs.CHAIN_URL }}
-      DEPLOY_BRANCH: ${{ vars.DEPLOY_BRANCH || github.event.inputs.DEPLOY_BRANCH }}
+      CHAIN_URL: ${{ github.event.inputs.CHAIN_URL || vars.CHAIN_URL }}
```

Comment:
This looks like a fallback. In which scenario would we expect it to get the value from `github.event.inputs` as opposed to `vars`?

Reply:
When trying to run the workflow manually
![Screenshot 2024-07-26 at 1 55 08 PM](https://github.com/user-attachments/assets/6e92b907-a520-432a-bb3d-9d1dcc6a15eb)